R/path_utils.R

```diff
@@ -133,6 +133,16 @@ split_path <- function(dirs, prefix) {
   dirs[nzchar(dirs)]
 }
 
+# https://github.com/r-lib/lintr/pull/2613
+# Simple wrapper around normalizePath but ensures forward slash on Windows
+normalize_path <- function(path, winslash = "/", mustWork = NA) { # nolint: object_name_linter
```

Comment:
I would simply drop `winslash=` from the signature, we don't have any plans of ever using it right?

Reply:
Good idea! Indeed, but could always be added back if needed.

Reply:
i think if needed you would just use `normalizePath()` directly :)